deployer/scripts/upgrade.sh

```diff
@@ -513,6 +514,14 @@ function regenerate_secrets_and_support_objects() {
   generate_support_objects
 }
 
+function remove_triggers() {
+
+  for dc in $(oc get dc -l logging-infra -o jsonpath='{.items[?(@.spec.triggers[*].type)].metadata.name}'); do
```

Comment:
doing the `?(@.spec.triggers[*].type)` check in the jsonpath makes it so it only returns dc that still have a trigger.  it's so that we aren't unnecessarily patching DC.